includes/services.inc.php

```diff
@@ -129,7 +129,7 @@ function poll_service($service)
 
     // If we do not have a cmd from the check script, build one.
     if ($check_cmd == "") {
-        $check_cmd = $config['nagios_plugins'] . "/check_" . $service['service_type'] . " -H " . ($service['service_ip'] ? $service['service_ip'] : $service['hostname']);
```

Comment:
The web ui shows the nagios checks that are located in your plugin dir, you are using check_ping here so you'd need a check_ping.inc.php. What params are you passing as you mush provide a -H value to check_ping anyway which is already taken care of.